src/Symfony/Component/DependencyInjection/Tests/ContainerTest.php

```diff
@@ -148,13 +148,27 @@ public function testSet()
         $this->assertSame($foo, $sc->get('foo'), '->set() sets a service');
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation Unsetting the "foo" service is deprecated since Symfony 3.3 and won't be supported anymore in Symfony 4.0.
+     * @expectedDeprecation Unsetting the "bar" service is deprecated since Symfony 3.3 and won't be supported anymore in Symfony 4.0.
```

Comment:
@nicolas-grekas tests pass with or without adding this latest `@expectedDeprecation`.. not sure thats a good sign?